packages/catalog-model/src/entity/Entity.ts

```diff
@@ -161,3 +166,23 @@ export type EntityRelationSpec = {
    */
   target: EntityName;
 };
+
+/**
+ * A link to external information that is related to the entity.
+ */
+export type EntityLink = {
```

Comment:
this should be exported in `index.ts` too

Reply:
Added